### PR TITLE
Fix #1908: Tags amount

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] Docs in [NexT website](https://theme-next.org/docs/) have been added / updated (for new features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue resolved: https://github.com/iissnan/hexo-theme-next/issues/1908

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

File: layout/page.swig [hunk 1]
```diff
@@ -43,9 +43,9 @@
             </div>
             <div class="tag-cloud-tags">
               {% if not theme.tagcloud %}
-                {{ tagcloud({min_font: 12, max_font: 30, amount: 300, color: true, start_color: '#ccc', end_color: '#111'}) }}
+                {{ tagcloud({min_font: 12, max_font: 30, amount: 200, color: true, start_color: '#ccc', end_color: '#111'}) }}
```
Comment (JiangTJ) [inline layout/page.swig]: default 300 -> 200?

**Comment by 1v9** [comment]
:laughing: why 200 -> 300 -> 200?

**Comment by stevenjoezhang** [comment]
@1v9 @JiangTJ https://github.com/iissnan/hexo-theme-next/commit/de271c1785166d1713fe4f89afffd853c43cc03d, "Temporary" solution for #1908.

**Comment by 1v9** [comment]
I think keep it, or add a statement for tagcloud amount, `# if your tags more than 200, you should enable this option and change amount value`

**Comment by stevenjoezhang** [comment]
I'll add it to the docs in theme-next.org

**Pull request merged**